addon/components/pix-select.js

```diff
@@ -158,7 +158,9 @@ export default class PixSelect extends Component {
 
   @action
   setSelectWidth() {
-    const baseFontRemRatio = 16;
+    const baseFontRemRatio = Number(
+      getComputedStyle(document.querySelector('html')).fontSize.match(/\d+/)[0]
```

Comment:
Ne fonctionne pas avec les nombres à virgule 😅
```suggestion
      getComputedStyle(document.querySelector('html')).fontSize.match(/\d+(\.\d+)?/)[0]
```